api/src/AppBundle/Entity/User.php

```diff
@@ -124,7 +124,7 @@ class User implements UserInterface
     /**
      * @var string
      * @ORM\Column(name="password", type="string", length=100, nullable=false)
-     * @JMS\Exclude
+     * @JMS\Groups({ "user-login"})
      */
     private $password;
```

Comment:
Without returning the password, the `AbstractToken` on the client side could never detect that it had changed, thus preventing the automatic redirect to login from working.